index.html

```diff
@@ -0,0 +1,460 @@
+</head>
+<body>
+    <section>
+        <nav id="navbar" class="navbar fixed-top navbar-expand-lg navbar-dark bg-secondary">
+          <div class="container-fluid">
+            <a class="navbar-brand" href="#">John Doe</a>
+            <button class="navbar-toggler" 
+              type="button" 
+              data-bs-toggle="collapse" 
+              data-bs-target="#navbarNavAltMarkup"
+              aria-controls="navbarNavAltMarkup"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+              <div class="navbar-nav">
+                <a class="nav-link text-white" href="#portfolio">Portfolio</a>
+                <a class="nav-link text-white" href="#about">About</a>
+                <a class="nav-link text-white" href="#contact">Contact</a>
+                <i class="bi bi-envelope text-white"></i>
+              </div>
+            </div>
+          </div>
+        </nav>
+    </section>
+      <!-- HOME SECTION -->
+      <section class="container-fluid bg-dark pt-5 pt-lg-0">
+        <div class="row py-5 py-lg-2">
+          <div class="col-lg-1 d-flex align-items-center order-3 order-lg-0">
+            <ul class="tags flex-lg-column flex-md-row">
+              <li class="elements">
+                <a href="#">
+                  <i class="bi bi-twitter text-white"></i>
+                </a>
+              </li>
+              <li class="mb-2 me-2">
+                <a href="#">
+                  <i class="bi bi-linkedin text-white"></i>
+                </a>
+              </li>
+              <li class="mb-2 me-2">
+                <a href="#">
+                  <i class="bi bi-instagram text-white"></i>
+                </a>
+              </li>
+              <li class="mb-2 me-2">
+                <a href="#">
+                  <i class="bi bi-github text-white"></i>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="col-lg-5 text-white d-flex flex-column justify-content-center">
+            <h1 class="text-danger h1 mb-3">
+              Hey there.<br />
+              I’m Alex
+            </h1>
+            <h2 class="mb-3">I’m a software developer</h2>
+            <p class="card-text">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have a
+              project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+          <div class="col-lg-6 d-flex justify-content-end d-none d-lg-block">
+            <img
+              src="images/Illustration-header@2x.png"
+              class="img-fluid"
+              alt="home section decoration black and white image"
+            />
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly add the background image for this section using CSS and not img tags, see screenshot above 👍🏿 
- [ ] This section should have the margins (top, bottom, left, right) defined in the Figma design, please implement this 👍🏿